File: solutions/001/001.js
function sum(n, k) {
  return Math.trunc(k * (Math.trunc((n - 1) / k) * (Math.trunc((n - 1) / k) + 1)) / 2)
}

const solution = sum(1000, 3) + sum(1000, 5) - sum(1000, 15)
console.log(solution)

File: solutions/001/_001.cs
using System;

public class _001 {
    public static void Main() {
        Console.WriteLine(Sum(1000, 3) + Sum(1000, 5) - Sum(1000, 15));
    }

    public static int Sum(int n, int k) {
        return k * ((((n - 1) / k) * (((n - 1) / k) + 1)) / 2);
    }
}
